### ISSUE TITLE
we are not getting a 2year extension these days

### ISSUE THREAD

**Issue by UjjwalSharma01** [Lomusire/unlimited-cloud-storage#2]
from the past 2 days I'm not getting 2-year extension on any of my account, so this limit has been engaged by terabox or what?

**Comment by Lomusire** [comment]
No TeraBox sadly just has a limit of 20 year premium at a time however you
still can keep collecting coins and premium everyday will be added to the
limit of 20 year from.now

Ujjwal Sharma ***@***.***> schrieb am Fr., 27. Sep. 2024,
18:58:

**Comment by Lomusire** [comment]
Did you run the extension till it said comeback tomorrow? It should still be 2 year premium if you not yet reached the limit

**Comment by UjjwalSharma01** [comment]
Yes, I ran it until the 'comeback tomorrow' popup and haven’t reached the limit of 20 years. As of now, it is current until 2037.

**Comment by Yoyo102020** [comment]
I think the amount of storage that could be auto claimed (using the extension) has also been reduced.

**Comment by UjjwalSharma01** [comment]
Both are Direct proportional i guess as far as i have observed

**Comment by Lomusire** [comment]
OK then they maybythey  reduced rewards for some country's sorry for that sadly can't do anything about this (except adding 80 extra coins for the other game requests wich is not Realy worth it) atleast it still is pretty good

**Comment by The-GO** [comment]
Free storage has not increased, I only got 3 months of membership per day

**Comment by Lomusire** [comment]
Yea terabox sadly decreased limit but you can still redeem extra storage
using the terabox mobile app

ku ***@***.***> schrieb am Mo., 30. Sep. 2024, 20:28:
